packages/lib/time.ts

```diff
@@ -90,7 +91,8 @@ class Time {
 	}
 
 	public unixMsToLocalDateTime(ms: number): string {
-		return moment.unix(ms / 1000).format('DD/MM/YYYY HH:mm');
+		const format = `${Setting.value("dateFormat")} ${Setting.value("timeFormat")}`;
```

Comment:
This lib shouldn't have a dependency to the settings. I think you just need to use one of the proper function